### ISSUE TITLE
Preview audio continues even after exiting app

### ISSUE THREAD

**Issue by dgw** [Piranhaplant/LLP2ST#2]
Steps:
- Open LLP2ST
- Tap a song in the list to expand the options
- Tap Preview
- Wait for preview to begin playing
- Hit system Back button until LLP2ST quits

Expected result: Audio stops playing when LLP2ST quits.
Actual result: Audio continues until LLP2ST is swiped out of the recent apps.

**Comment by Piranhaplant** [comment]
Yeah, I knew about this issue, but I don't know how to fix it since Android doesn't provide any way to tell when the user leaves the whole application. I can detect when the user leaves the current activity, but I want the audio to play on all activities in the app.

My idea was to have a notification when previewing a song (similar to most Android music players), but I haven't had time to implement it yet.

**Comment by dgw** [comment]
It does seem to be possible. SIF itself does not continue playing menu background music if one presses the Home button. Though it may be moot to compare LLP2ST, as SIF very well might use only one Activity to host the game engine.

**Comment by Piranhaplant** [comment]
SIF (and most other games) indeed only use one activity, so it is easy for them to detect this. I'll try some other stuff, but I think using a notification is the best option.
